sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 12.8.0-beta.3 (Unreleased)
+## 12.8.0 (2023-09-12)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Features in `12.8.0-beta.1` and `12.8.0-beta.2` are now generally available.
```

Comment:
Same here, keep this under some sub-heading. If not `Features Added` (if you feel that isn't appropriate), then `Other Changes` might be a good alternative.

The previous betas did have new features, so `Features Added` makes sense.